java/ql/src/Security/CWE/CWE-022/TaintedPath.qhelp

```diff
@@ -36,15 +38,23 @@ the user input matches one of these patterns.</p>
 
 <p>In this example, a file name is read from a <code>java.net.Socket</code> and then used to access a file
 and send it back over the socket. However, a malicious user could enter a file name anywhere on the file system,
-such as "/etc/passwd".</p>
+such as "/etc/passwd" or "../../../etc/passwd".</p>
 
-<sample src="TaintedPath.java" />
+<sample src="examples/TaintedPath.java" />
 
-<p>Simply checking that the path is under a trusted location (such as a known public folder) is not enough,
-however, since the path could contain relative components such as "..". To fix this, check that it does
-not contain ".." and starts with the public folder.</p>
+<p>
+If the input is just supposed to be a file name, you can check that it doesn't contain any path separators
+or ".." sequences.
```

Comment:
```suggestion
If the input should only be a file name, you can check that it doesn't contain any path separators or ".." sequences.
```

V minor: suggestion to change "just supposed to be" to "should be", to make it slightly less idiomatic (ie "supposed to be" might be tricky for some non-native English speakers)